ssl/ssl_cipher.cc

```diff
@@ -1446,6 +1446,17 @@ const SSL_CIPHER *SSL_get_cipher_by_value(uint16_t value) {
       ssl_cipher_id_cmp));
 }
 
+const SSL_CIPHER *SSL_CIPHER_find(SSL *ssl, const unsigned char *ptr) {
```

Comment:
Is this supposed to use the ssl parameter for anything? Like only return the matching cipher if it's enabled in ssl?

Reply:
In OpenSSL, the get cipher by value function is stored in ssl->method. For AWS-LC, we have a function defined separately which SSL_CIPHER_find is wrapping. So in our case, it's essentially just there because of the function signature.

Reply:
Even though we don't use the `ssl` parameter, should we still require that it not be null to provide behavior more similar to OpenSSL?

Reply:
added a condition to check